app/src/main/java/com/practicum/playlistmaker/library/ui/fragments/MediaLibraryViewPagerAdapter.kt

```diff
@@ -0,0 +1,18 @@
+package com.practicum.playlistmaker.library.ui.fragments
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class MediaLibraryViewPagerAdapter(activity: AppCompatActivity) : FragmentStateAdapter(activity) {
+
+    override fun getItemCount(): Int = 2
+
+    override fun createFragment(position: Int): Fragment {
+        return when (position) {
+            0 -> PlaylistsFragment.newInstance()
+            1 -> FavouriteTracksFragment.newInstance()
+            else -> throw IllegalArgumentException("Invalid position: $position")
```

Comment:
Лучше эксепшн не выбрасывать тут, так как он нигде не обрабатывается и приведёт к крешу приложения. Лучше написать так:
```suggestion
            else -> FavouriteTracksFragment.newInstance()
```